### DISCUSSION TITLE
Missing Plotting Functions in Tigramite Documentation?

### DISCUSSION THREAD

**Discussion by ZackKaufman** [jakobrunge/tigramite#234] [Q&A]
Hello Tigramite Developers, 

The documentation site for the latest release of Tigramite (https://jakobrunge.github.io/tigramite/) appears to be missing the descriptions for plotting functions, which were shown in earlier releases. I am trying to re-format a figure built using tp.plot_graph for a presentation this week, but need to be able to reference the parameter name conventions to do so. 

Would anyone be able to provide access to documentation for tp.plot_graph (new or old)? 

Thank you, 
Zack Kaufman, UCSC climate dynamics group.

**Comment by Master-Chandler** [comment]
I suffer the same problem. I also need to adjust my figures.

**Comment by jakobrunge** [comment]
They should be there now. You can always directly read the docstrings.
